tcn-client-android/src/main/java/org/tcncoalition/tcnclient/crypto/Serialization.kt

```diff
@@ -0,0 +1,25 @@
+package org.tcncoalition.tcnclient.crypto
+
+import java.nio.ByteBuffer
+
+/** Convenience function to read a 32-byte array. */
+internal fun read32(buf: ByteBuffer): ByteArray {
```

Comment:
The class name is probably is not suited anymore here as we cannot do extensions of classes with interfaces as in Rust or Swift. 

We could rename it to `Extensions` and do something like this:
```
fun ByteBuffer.read32() = get(ByteArray(32))
```